src/ui/buttons/styles/button.js

```diff
@@ -125,4 +125,27 @@ export const buttonStyle = `
     .${ CLASS.CONTAINER } .${ CLASS.VAULT_HEADER } {
         margin-top: 10px;
     }
+
+    .${ CLASS.BUTTON }.${ CLASS.COLOR }-${ BUTTON_COLOR.BLACK } {
+        color: ${ COLOR.BLACK };
+    }
+
+    .${ CLASS.BUTTON }.${ CLASS.COLOR }-${ BUTTON_COLOR.WHITE } {
+        color: ${ COLOR.WHITE };
+    }
+
+    .${ CLASS.BUTTON }[${ ATTRIBUTE.FUNDING_SOURCE }=${ FUNDING.CARD }] * {
+        height: 1.2em;
+    }
+
+    .${ CLASS.BUTTON }[${ ATTRIBUTE.FUNDING_SOURCE }=${ FUNDING.CARD }] .${ CLASS.TEXT } {
+        position: absolute;
+        visibility: hidden;
+        font-weight: 400;
+    }
+
+    .${ CLASS.BUTTON }[${ ATTRIBUTE.FUNDING_SOURCE }=${ FUNDING.CARD }] {
+        background: transparent;
+        font-size: 1.2em;
```

Comment:
Can we keep the font size consistent with the other buttons?

Reply:
I wanted the text size to be relative to the size of the button since the text is quite long and doesn't look good for the buttons with small size.

Reply:
I also will double check if you are already using `em` for the font size.

Reply:
The default 14px font size doesn't look great for the black button text. Also, it doesn't get adjusted when the button size shrinks.

<img width="558" alt="variants htm 2019-12-13 15-04-49" src="https://user-images.githubusercontent.com/3213579/70837751-f7193100-1db9-11ea-8d7d-52ca772776ae.png">

Reply:
Which screenshot is which here?

Reply:
![70837751-f7193100-1db9-11ea-8d7d-52ca772776ae png (1116×1888) 2019-12-16 14-32-13](https://user-images.githubusercontent.com/3213579/70948750-fcbc8400-2010-11ea-99d0-b8b7964a36b9.png)